data/src/main/java/com/architect/coders/mu8/data/DummyClass.kt

```diff
@@ -0,0 +1,3 @@
+package com.architect.coders.mu8.data
+
+data class DummyClass (val name: String)
```

Comment:
Add a new line to the end of the file.

Reply:
Actually, let's remove this model if it is not needed, instead of leaving this you can add to the PR description what the reviewers needs to add in order to test this integration with data, like creating this object (that you can provide on the same description) and adding the dependency in the app gradle.